src/main/java/org/jenkinsci/plugins/ghprb/GhprbBuilds.java

```diff
@@ -59,6 +59,7 @@ public void build(GhprbPullRequest pr, GHUser triggerSender, String commentBody)
 
 
     public void onStarted(AbstractBuild<?, ?> build, TaskListener listener) {
+        GhprbCustomStatusRepoPasser.addRepo(build.getFullDisplayName(), repo.getGitHubRepo());
```

Comment:
So right now this is a memory leak.  Can we remove the repo from the list when the build is completed, or when is it safe to be removed?